src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,55 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.operation.BalaceOperationHandler;
+import core.basesyntax.operation.OperationHandler;
+import core.basesyntax.operation.PurchaseOperationHandler;
+import core.basesyntax.operation.ReturnOperationHandler;
+import core.basesyntax.operation.SupplyOperationHandler;
+import core.basesyntax.service.FileCreationService;
+import core.basesyntax.service.ParseService;
+import core.basesyntax.service.ReaderService;
+import core.basesyntax.service.ReportService;
+import core.basesyntax.service.WriterService;
+import core.basesyntax.service.impl.FileCreationServiceImpl;
+import core.basesyntax.service.impl.ParserServiceImpl;
+import core.basesyntax.service.impl.ReaderServiceImpl;
+import core.basesyntax.service.impl.ReportServiceImpl;
+import core.basesyntax.service.impl.WriterServiceImpl;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
 public class HelloWorld {
+    private static final String FILE_WITH_DAY_ACTIVITIES = "src/main/resources/outputReport.csv";
+    private static final String REPORT_FILE = "src/main/resources/report.csv";
```

Comment:
I would like you to push both these files to Github - input file and output file